tests/unit/modules/dcnm/fixtures/dcnm_links_payloads.json

```diff
@@ -328,7 +328,7 @@
         "link-uuid": "LINK-UUID-1446600",
         "sw2-info": {
           "fabric-name": "mmudigon-numbered",
-          "if-name": "Ethernet1/1",
+          "if-name": "ethernet1/1",
```

Comment:
This is the mock response coming back from NDFC right?  Does it come back as upper case or lower case?

Reply:
Yes. But whether it is upper or lower depends on the switch model. Code is now updated to handle both cases appropriately. To check that it does, I have changes existing test cases to have both upper and lower cases for interface names